### PR TITLE
Make the explainer an HTML table

### PR DESCRIPTION
**Pull request by coolreader18**
I also did some optimizations for tablets so that the content isn't 25% width like it would be on desktops; it was pretty scrunched. Also also, I changed the explainer text and added some text relating to our progress the goals that we had, that we can eventually add to those boxes.

### CODE REVIEW

**Comment by coolreader18** [comment]
The HTML table looks better with the dark mode extension I have:
image

File: index.markdown [hunk 1]
```diff
@@ -3,7 +3,11 @@
 # To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
 
 layout: home
-explainer: "CPython is the common implementation of Python. We love C, but we love Rust a little more 😉... that's why we're working on a RustPython, a Python implementation written in Rust. We are doing it for fun, to learn about writing interpreters, and to have a useful, usable and embeddable implementation of Python in Rust."
+explainer:
```
Comment (mireille-raad) [inline index.markdown:6]: thank you for the re-write.

File: index.markdown [hunk 2]
```diff
@@ -19,8 +23,23 @@ installation:
   - command: "conda install rustpython -c conda-forge"
 
 goals:
-  - goal: "Full Python-3 environment entirely in Rust (not CPython bindings) with a clean implementation, without compatiblity hacks. "
-  - goal: "Fast, reliable and secure implementation of Python that can be used with Rust or compiled to WebAssembly. "
+  - goal:
+      Full Python-3 environment entirely in Rust (not CPython bindings), with
+      a clean implementation and no compatiblity hacks.
+    # TODO: integrate this into the "goals" boxes
+    progress:
```
Comment (mireille-raad) [inline index.markdown:30]: I like the progress boxes or text. Is the progress box supposed to show somewhere on the homepage?

**Comment by mireille-raad** [comment]
I think you can merge. 

My one comment is that it would be nice to see the "progress" boxes show up on the homepage. 

We can add those later maybe after we figure out some numbers for the progress.

**Review by mireille-raad — APPROVED** [review]

File: index.markdown (hunk 2 above)
Comment (coolreader18) [reply index.markdown:30]: I was thinking something like split the goals boxes into 2 vertically with the goal on top and progress on bottom:

image

(very rough sketch, of course)

Comment (mireille-raad) [reply index.markdown:30]: yeah that works :D 

If you want, you don't have to stick to the background I used. You can probably use borders or some form of highlight.

**Pull request merged**